test/ng/directive/ngModelSpec.js

```diff
@@ -578,6 +578,30 @@ describe('ngModel', function() {
 
         dealoc(form);
       }));
+
+
+      it('should set NaN as the $modelValue when an asyncValidator is present',
+        inject(function($q) {
+
+        ctrl.$asyncValidators.test = function() {
+          return $q(function(resolve, reject) {
+            resolve();
+          });
+        };
+
+        scope.$apply('value = 10');
+        expect(ctrl.$modelValue).toBe(10);
+
+        expect(function() {
+          scope.$apply(function() {
+            scope.value = NaN;
+          });
+        }).not.toThrowMinErr('$rootScope', 'infdig', '10 $digest() iterations reached. Aborting!\n' +
```

Comment:
I would prefer a plain `not.toThrow()`, because this test might become useless if the minErr changes.